app/src/main/java/org/openobservatory/ooniprobe/model/OONITests.java

```diff
@@ -18,4 +20,8 @@
     public static final String NOTIFICATION_SERVER_DEV = "https://registry.proteus.test.ooni.io";
     public static final String NOTIFICATION_SERVER_PROD = "https://registry.proteus.ooni.io";
     public static final String NOTIFICATION_SERVER = NOTIFICATION_SERVER_PROD;
+    public static final int VERBOSITY = LogSeverity.LOG_INFO;
```

Comment:
What is `VERBOSITY` used for? I find this name a bit weird

Reply:
it's the global MK verbosity